docker/api/cnaas-setup.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e
+set -x
+
+export DEBIAN_FRONTEND noninteractive
+
+/bin/sed -i s/deb.debian.org/ftp.se.debian.org/g /etc/apt/sources.list
```

Comment:
Think you can do something like: ENV REPO ftp.se.debian.org

Reply:
This is something I stole from the eduid project but it seems to be working quite well. Do you know how to do it with env variables instead? I didn't find anything on a quick google search